code/modules/surgery/organs/augments_internal.dm

```diff
@@ -457,6 +454,87 @@
 	if(H.stat == CONSCIOUS)
 		to_chat(H, "<span class='notice'>You feel your heart beating again!</span>")
 
+/obj/item/organ/internal/cyberimp/chest/ipc_repair
+	name = "Reactive Repair Implant"
+	desc = "This implant reworks the IPC frame, in order to incoperate materials that return to their original shape after being damaged. Requires power to function"
+	implant_color = "#0ac0d8"
+	origin_tech = "materials=4;programming=4;biotech=4;magnets=4;engineering=4"
+	slot = "stomach" //Can't have a nutriment pump with it.
+	requires_machine_person = TRUE
+
+/obj/item/organ/internal/cyberimp/chest/ipc_repair/on_life()
+	if(crit_fail)
+		return
+	if(owner.maxHealth == owner.health)
+		owner.adjust_nutrition(-0.5)
+		return //Passive damage scanning
+
+	owner.adjustBruteLoss(-0.5, robotic = TRUE)
+	owner.adjustFireLoss(-0.5, robotic = TRUE)
+	owner.adjust_nutrition(-4) //Very power inefficent. Hope you got an APC nearby.
+
+/obj/item/organ/internal/cyberimp/chest/ipc_repair/emp_act(severity)
+	if(!owner || emp_proof || crit_fail)
+		return
+	crit_fail = TRUE
+	addtimer(VARSET_CALLBACK(src, crit_fail, FALSE), 30 SECONDS / severity)
+
+/obj/item/organ/internal/cyberimp/chest/ipc_joints
+	name = "IPC ER-OR Joint Implant"
+	desc = "This is a basetype. Notify a coder!"
+	implant_color = "#eeff00"
+	origin_tech = "materials=5;programming=4;biotech=4"
+	slot = "joints"
+	requires_machine_person = TRUE
+
+/obj/item/organ/internal/cyberimp/chest/ipc_joints/magnetic_joints
+	name = "Magnetic Joints Implant"
+	desc = "This implant modifies IPC joints to use magnets, allowing easy re-attachment and fluid movement."
```

Comment:
over the design description you use "smooth" instead of "fluid".

Reply:
Felt fluid movement was better than fluid so I changed it, iirc.
Which do you prefer?

Reply:
I personally prefer fluid.

Reply:
done